site/content/docs/commands/deploy.en.md

```diff
@@ -18,6 +18,7 @@ This command is used to run either [`copilot svc deploy`](../commands/svc-deploy
 ```bash
   -a, --app string                     Name of the application.
   -e, --env string                     Name of the environment.
+      --force                          Optional. Force a new ECS service deployment using the existing image.
```

Comment:
Do we actually check if it's an ECS service when people run `copilot deploy --force` ?
What's the behavior if they use `--force` on an app runner service? I wonder if for App Runner we can call: https://docs.aws.amazon.com/apprunner/latest/api/API_StartDeployment.html

Reply:
Right now if they do `--force` for app runner service then nothing happen.,

Reply:
> I wonder if for App Runner we can call: https://docs.aws.amazon.com/apprunner/latest/api/API_StartDeployment.html

That's a good idea!

Reply:
Ok done. Added `--force` for app runner.